lambdas/src/apis/profiles/controllers/profiles.ts

```diff
@@ -83,7 +76,7 @@ function addBaseUrlToSnapshots(baseUrl: string, metadata: EntityMetadata): Entit
     return { avatars }
 }
 
-interface V3ControllerEntity {
+type V3ControllerEntity = {
```

Comment:
This type and `V3ControllerEntityContent` are also available on `dcl-catalyst-commons`. There is no need to re-define them

Reply:
Same here, this entity is enforcing the metadata type.